src/resources/Fields/FieldsInterfaces.ts

```diff
@@ -32,3 +34,17 @@ export interface ListFieldsParams extends Paginated {
     sortBy?: string;
     type?: FieldTypes;
 }
+
+export interface FieldListingFilters {
+    facet?: FacetOrSortStatus;
```

Comment:
Do you think we could add some description for each properties (from [doc](https://docs.coveo.com/en/8/api-reference/field-api#tag/Fields/operation/getFieldsUsingPOST_6))?